godtools/App/Features/Favorites/Data-DomainInterface/GetYourFavoritedToolsRepository.swift

```diff
@@ -16,16 +16,14 @@ class GetYourFavoritedToolsRepository: GetYourFavoritedToolsRepositoryInterface
     private let getTranslatedToolName: GetTranslatedToolName
     private let getTranslatedToolCategory: GetTranslatedToolCategory
     private let getToolListItemInterfaceStringsRepository: GetToolListItemInterfaceStringsRepository
-    private let getTranslatedToolLanguageAvailability: GetTranslatedToolLanguageAvailability
```

Comment:
Good call on removing the dependency, I think it aids in the visibility that favorited tools won't support language availability.